front/src/store/modules/ReristrationUser.store.js

```diff
@@ -0,0 +1,52 @@
+import axios from 'axios';
+
+export default {
+  actions: {
+    AddUser ({ commit }, newUser) {
+      axios({
+        method: 'post',
+        url: 'https://localhost:44333/api/user',
+        data: {
+          userName: newUser.userName,
+          email: newUser.email,
+          userPassword: newUser.userPassword
+        }
+      }).then(response => {
+        if (Number(response.data)) {
```

Comment:
it is really confusing
why we check that response is number.

See, response from server it is status code which means http status of success and unsuccessful with some types of error. and data. 

I sure you should change the model of response. If you want to return unsuccessful result it should be something like {isSuccess: false, isEmailExist: true} and handle it in component. Or other option, server return error and error status specific for unsuccessful registration you check it in store and return in promise {isSuccess: false, isEmailExist: true} and component handles it